packages/babel-plugin-transform-es2015-modules-umd/src/index.js

```diff
@@ -47,6 +47,7 @@ export default function ({ types: t }) {
           let moduleName = args.length === 3 ? args.shift() : null;
           let amdArgs = call.arguments[0];
           let func = call.arguments[1];
+          let browserGlobals = arguments[1].opts && arguments[1].opts.browserGlobals || {};
```

Comment:
I thought you meant some already defined variable I wasn't aware of.

On a side note, there's no need to provide this kind of answers. From the experience (jQuery team), if you start a reply with "Just do [PUT SENTENCE HERE]", not only it isn't useful for the contributor but you also sound a lot like a... you can guess what. While I'm able to understand what you mean (I'm definitely not a beginner), this kind of answers are very discouraging and embarrassing for beginners. They feel they are missing something obvious and there are high chances they will not contribute again.